<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result_deposit</class>
 <widget class="QWidget" name="Result_deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_sum_proc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма на вкладе</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Месяц</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Начислено процентов</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_sum_dep">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_nalog">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    qproperty-alignment: 'AlignCenter';
}

font: 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
